htdocs/core/modules/facture/doc/pdf_crabe.modules.php

```diff
@@ -357,6 +357,20 @@ public function write_file($object, $outputlangs, $srctemplatepath = '', $hidede
 				$pdf->SetKeyWords($outputlangs->convToOutputCharset($object->ref)." ".$outputlangs->transnoentities("PdfInvoiceTitle")." ".$outputlangs->convToOutputCharset($object->thirdparty->name));
 				if (! empty($conf->global->MAIN_DISABLE_PDF_COMPRESSION)) $pdf->SetCompression(false);
 
+				$cert=file_get_contents(DOL_DATA_ROOT."/users/".$user->id."/certificates/signature.crt");
```

Comment:
A crt is a text string. So why not just setting the crt text into a field stored into llx_user table ?